data/openscap/oscap-dict.patch

```diff
@@ -0,0 +1,14 @@
+--- /usr/share/openscap/cpe/openscap-cpe-dict.xml	2022-05-07 21:11:24.000000000 -0400
++++ /usr/share/openscap/cpe/openscap-cpe-dict.xml	2022-09-15 07:30:17.067140391 -0400
+@@ -65,6 +65,10 @@
+             <title xml:lang="en-us">openSUSE Leap 15.3</title>
+             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.opensuse:def:153</check>
+       </cpe-item>
++      <cpe-item name="cpe:/o:opensuse:leap:15.5">
++            <title xml:lang="en-us">openSUSE Leap 15.5</title>
++            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.opensuse:def:155</check>
++      </cpe-item>
```

Comment:
Hm, if I understand correctly, the lines that you add manually for leap 15.5 should be fixed in a future update, right? 
If yes, I disagree on doing this workaround. We are in product testing and these failures, because of a bug should be visible :)